README.md

```diff
@@ -0,0 +1,41 @@
+# Hansken Kaitai Plugin Template for Python
+
+This repository contains a template for a Hansken Kaitai plugin written in Python.
+The template contains the general structure of a Kaitai plugin, including the required build steps.
+
+To get started developing Hansken Extraction Plugins, read the [Getting Started Documentation](https://netherlandsforensicinstitute.github.io/hansken-extraction-plugin-sdk-documentation/latest/dev/python/getting_started.html).
+Furthermore, it is strongly recommended to take note of the [Hansken Extraction Plugins for plugin developers documentation](https://netherlandsforensicinstitute.github.io/hansken-extraction-plugin-sdk-documentation/latest/).
+
+The implementer of this template, is required to conduct a few general steps to enable parsing the contents of a specific filetype to a JSON, as is explained in more detail below.
+The template contains utility in [`kaitai_utils.py`](kaitai_utils.py). This utility python file, which only contains 'static' methods, contains the generic parse methods. 
+The implementer of the new Kaitai plugin, does not need to touch this `kaitai_utils.py` file. 
```

Comment:
If the user does not need to touch kaita_utils.py, I think the documentation on it can be left out. Now I don't know why I just read this.

Reply:
@luuk-nfi what makes you confused? I wanted to explain the contents of the template in a very general way to make things clearer, but if it leaves you confused, this is not the right way. You suggest to leave any mention about `kaitai_utils.py` completely?